test/core/end2end/tests/no_logging.cc

```diff
@@ -139,6 +139,12 @@ void SimpleRequest(CoreEnd2endTest& test) {
 }
 
 CORE_END2END_TEST(NoLoggingTest, NoLoggingTest) {
+// TODO(hork): remove when the listener flake is identified
+#ifdef GPR_WINDOWS
+  if (IsEventEngineListenerEnabled()) {
+    return;
```

Comment:
GTEST_SKIP() << "not for windows + event engine listener";

would probably be preferred

Reply:
Fixed.